app/components/nav-bar/template.hbs

```diff
@@ -0,0 +1,23 @@
+
+<div class="nav-item-group breadcrumbs">
```

Comment:
Clicking on different parts of the breadcrumb gives me different errors (probably from routing issues)

Reply:
None of breadcrumb pieces actually seem to transition me though

Reply:
I'll double check. I seems to work on my local.